modules/reactor/src/meshgenerators/PolygonMeshGeneratorBase.C

```diff
@@ -1738,3 +1739,30 @@ PolygonMeshGeneratorBase::modifiedSingleBdryLayerParamsCreator(
   mod_single_bdry_layer_params.bias = std::pow(mod_single_bdry_layer_params.bias, 1.0 / order);
   return mod_single_bdry_layer_params;
 }
+
+std::string
+PolygonMeshGeneratorBase::pitchMetaDataErrorGenerator(
+    const std::vector<MeshGeneratorName> & input_names,
+    const std::vector<Real> & metadata_vals,
+    const std::string & metadata_name) const
+{
+  const unsigned int input_name_size =
+      std::max_element(input_names.begin(),
+                       input_names.end(),
+                       [](const auto & a, const auto & b) { return a.size() < b.size(); })
+          ->size();
+  std::stringstream detailed_error;
+  detailed_error << std::endl;
+  detailed_error << std::string(input_name_size + metadata_name.size() + 7, '-') << std::endl;
+  detailed_error << std::fixed << std::left << std::setw(input_name_size + 4)
+                 << std::setprecision(6) << "input name"
+                 << "|" << std::fixed << std::right << std::setw(metadata_name.size() + 2)
+                 << metadata_name << std::endl;
+  detailed_error << std::string(input_name_size + metadata_name.size() + 7, '-') << std::endl;
+  for (unsigned int i = 0; i < input_names.size(); i++)
+    detailed_error << std::fixed << std::left << std::setw(input_name_size + 4) << input_names[i]
+                   << "|" << std::fixed << std::right << std::setw(metadata_name.size() + 2)
+                   << metadata_vals[i] << std::endl;
+  detailed_error << std::string(input_name_size + metadata_name.size() + 7, '-') << std::endl;
```

Comment:
there is a utility to print tables that are formatted right, see postprocessors table output for example, FormattedTable

I dont see what you need there to output it as a stringstream though

Reply:
I will check that, thanks.

Reply:
FormattedTable works but it output extra `time` column. It has a `_output_time` attribute but it only works for CSV output. Is there an existing way to avoid that time column or I should modify FormattedTable to add the option?

Reply:
use this API to turn it off?
```
  /**
   * Set whether or not to output time column.
   */
  void outputTimeColumn(bool output_time) { _output_time = output_time; }
```

Reply:
Yes, it only works for CSV output. `printTable()` ignores this.

Reply:
ah my bad. Well then FormattedTable needs to be fixed to obey it